webots_ros2_universal_robot/launch/moveit_demo_launch.py

```diff
@@ -38,49 +39,63 @@ def load_file(filename):
     def load_yaml(filename):
         return yaml.safe_load(load_file(filename))
 
-    description = {'robot_description': load_file('moveit_ur5e_description.urdf')}
-    description_semantic = {'robot_description_semantic': load_file('moveit_ur5e.srdf')}
-    description_kinematics = {'robot_description_kinematics': load_yaml('moveit_kinematics.yaml')}
-    movegroup = {'move_group': load_yaml('moveit_movegroup.yaml')}
-    moveit_controllers = {
-        'moveit_controller_manager': 'moveit_simple_controller_manager/MoveItSimpleControllerManager',
-        'moveit_simple_controller_manager': load_yaml('moveit_controllers.yaml')
-    }
-    sim_time = {'use_sim_time': True}
-    run_move_group_node = Node(
-        package='moveit_ros_move_group',
-        executable='move_group',
-        output='screen',
-        parameters=[
-            description,
-            description_semantic,
-            description_kinematics,
-            moveit_controllers,
-            movegroup,
-            sim_time
-        ],
-    )
-
-    rviz_config_file = os.path.join(package_dir, 'resource', 'moveit_visualization.rviz')
-    rviz_node = Node(
-        package='rviz2',
-        executable='rviz2',
-        name='rviz2',
-        arguments=['-d', rviz_config_file],
-        parameters=[
-            description,
-            description_semantic,
-            description_kinematics,
-            sim_time
-        ],
-    )
-
-    robot = IncludeLaunchDescription(
-        PythonLaunchDescriptionSource(os.path.join(package_dir, 'launch', 'robot_launch.py'))
-    )
-
-    return LaunchDescription([
-        run_move_group_node,
-        rviz_node,
-        robot
-    ])
+    # Check if nav2_bringup is installed
```

Comment:
Shouldn't it be moveit here?
```suggestion
    # Check if moveit is installed
```